Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -79,15 +128,28 @@
     'vtPositives': 10,
     'vtPositiveUrlsForIP': 30
 }
-# The dictionary below does not represent DBot Scores correctly, and should not be used
-dbotscores = {
-    'Critical': 4,
-    'High': 3,
-    'Medium': 2,
-    'Low': 1,
-    'Unknown': 0,
-    'Informational': 0.5
-}
+
+
+class DBotScoreType(object):
+    """ ignore docstring
+    Enum: contains all the indicator types, used DBotScore.Type)
+    """
+    IP = 'ip'
+    HASH = 'hash'
```

Comment:
Pretty sure that we should mark it as a file rather then hash
```suggestion
    FILE = 'file'
```